deploy/template.yaml

```diff
@@ -357,358 +252,77 @@ Resources:
           Value: backend
         - Key: Name
           Value: APIGatewayAccessLogGroup
-        # - Key: Source
-        #   Value: alphagov/di-devplatform-demo-sam-app/sam-app/template.yaml
 
-  HelloWorldRestApi:
-    Type: AWS::Serverless::Api
-    Properties:
-      StageName: Prod
-      AccessLogSetting:
-        Format: "$context.requestId $context.httpMethod $context.path"
-        DestinationArn: !GetAtt APIGatewayAccessLogGroup.Arn
-      CacheClusterEnabled: true
-      CacheClusterSize: "0.5"
-      DefinitionBody:
-        Fn::Transform:
-          Name: AWS::Include
-          Parameters:
-            Location: './spec/private-api.yaml'
-      TracingEnabled: true
-      Tags:
-        Product: GOV.UK Sign In
-        System: Dev Platform
-        Environment: Demo
-        Service: backend
-        Name: HelloWorldRestApi
-        Source: alphagov/di-devplatform-demo-sam-app/sam-app/template.yaml
+  ### End of API Gateway definition.
 
-  HelloWorldFunction:
+  ### Function Definition
+
+  # ClaimedIdentity
+  ClaimedIdentityFunction:
     Type: AWS::Serverless::Function
+    DependsOn: ClaimedIdentityFunctionLogGroup
     Properties:
-      Handler: HelloWorldHandler.lambdaHandler
+      FunctionName: !Sub "CIC-ClaimedIdentity-${AWS::StackName}"
+      Handler: ClaimedIdentityHandler.lambdaHandler
       CodeUri: ../src/
-      Runtime: nodejs16.x
+      Runtime: nodejs18.x
```

Comment:
Our entire code source is currently one repo. Could we move this to a globally defined property for all lambdas?